### ISSUE TITLE
origina code pretrain model can use in this code?

### ISSUE THREAD

**Issue by zdx3578** [jazzsaxmafia/Weakly_detector#1]
https://github.com/metalbubble/CAM  this origina code pretrain model can use in this code?

**Comment by jazzsaxmafia** [comment]
No. Unfortunately you can't. Let me know if you need trained model file.

**Comment by zdx3578** [comment]
how to run? have a demo example? where to download your model and date dir in gitignore?

**Comment by cemsaz** [comment]
How can we run this with a pre-trained model? I don't have a GPU :( @jazzsaxmafia Can you provide one?

**Comment by SafwanMahmood** [comment]
Can you please provide a trained model file and guide me on how to pre-train

**Comment by Rahul-Venugopal** [comment]
Is there a trained model file available ? @jazzsaxmafia  @SafwanMahmood
